src/picongpu/include/simulationControl/MySimulation.hpp

```diff
@@ -332,6 +333,9 @@ class MySimulation : public SimulationHelper<simDim>
 
         /* add CUDA streams to the StreamController for concurrent execution */
         Environment<>::get().StreamController().addStreams(6);
+
+        // Initialize synchrotron functions
+        this->synchrotronFunctions.init();
```

Comment:
Please avoid that synchrotron function creates memory if we not calculate/use them.